src/Iter.mo

```diff
@@ -43,13 +44,14 @@ module {
 
   /// Calls a function `f` on every value produced by an iterator and discards
   /// the results. If you're looking to keep these results use
-  /// [`map`](#value.map).
-  /// ```
+  /// <<value.map>>.
```

Comment:
> Why do we need to convert Markdown to asciidoc in mo-doc, instead of using asciidoc directly in the comments?

Asciidoc is something we use in DFINITY's infrastructure, and we have our own elaborate setup to make it work with Antora, but our docs site is not the only place people will want to read documentation for Motoko libraries. See eg: https://kritzcreek.github.io/motoko-matchers/

I don't think it's a good default to impose the relatively obscure `Asciidoc` on every Motoko developer. Also if this was `Asciidoc` I'd have to write an OCaml parser for the format.

Reply:
I see. Then we should probably run `mo-doc` as part of the CI here. So your suggestion is to revert this change and fix in the `mo-doc` binary, right?

Reply:
Done

Reply:
My plan was actually to just remove all links and turn them into plain text references to functions the users has to figure out themselves until we add the feature to `mo-doc`. But I can do that in a follow-up PR, so it doesn't block this one.